drivers/spi/spi_nrfx_spim.c

```diff
@@ -73,7 +74,7 @@ struct spi_nrfx_data {
 	uint8_t ppi_ch;
 	uint8_t gpiote_ch;
 #endif
-#ifdef USE_CLOCK_REQUESTS
+#if USE_CLOCK_REQUESTS
```

Comment:
Is this actually worth it? `SPI_BUFFER_IN_RAM` still uses `#ifdef` and that can potentially lead to confusion/mistakes.